{"name": "Login Authentication, single paramters", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div/div/div[2]/nav/div[3]/h1\"}\n  (Session info: chrome=114.0.5735.248); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x0028A813+48355]\n\t(No symbol) [0x0021C4B1]\n\t(No symbol) [0x00125358]\n\t(No symbol) [0x001509A5]\n\t(No symbol) [0x00150B3B]\n\t(No symbol) [0x0017E232]\n\t(No symbol) [0x0016A784]\n\t(No symbol) [0x0017C922]\n\t(No symbol) [0x0016A536]\n\t(No symbol) [0x001482DC]\n\t(No symbol) [0x001493DD]\n\tGetHandleVerifier [0x004EAABD+2539405]\n\tGetHandleVerifier [0x0052A78F+2800735]\n\tGetHandleVerifier [0x0052456C+2775612]\n\tGetHandleVerifier [0x003151E0+616112]\n\t(No symbol) [0x00225F8C]\n\t(No symbol) [0x00222328]\n\t(No symbol) [0x0022240B]\n\t(No symbol) [0x00214FF7]\n\tBaseThreadInitThunk [0x76B87D59+25]\n\tRtlInitializeExceptionChain [0x77DCB79B+107]\n\tRtlClearBits [0x77DCB71F+191]\n\t(No symbol) [0x00000000]\n\n", "trace": "  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\loginsteps.py\", line 27, in step_impl\n    text = context.driver.find_element(By.XPATH,\"/html/body/div/div/div[2]/nav/div[3]/h1\").text\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Launch chrome browser", "status": "passed", "start": 1690138043737, "stop": 1690138045193}, {"name": "When Open login page", "status": "passed", "start": 1690138045193, "stop": 1690138046419}, {"name": "And Enter phone \"0208033577\" and pin \"1996\"", "status": "passed", "start": 1690138046419, "stop": 1690138046688}, {"name": "And Click on Enter", "status": "passed", "start": 1690138046689, "stop": 1690138046760}, {"name": "Then successful login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div/div/div[2]/nav/div[3]/h1\"}\n  (Session info: chrome=114.0.5735.248); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x0028A813+48355]\n\t(No symbol) [0x0021C4B1]\n\t(No symbol) [0x00125358]\n\t(No symbol) [0x001509A5]\n\t(No symbol) [0x00150B3B]\n\t(No symbol) [0x0017E232]\n\t(No symbol) [0x0016A784]\n\t(No symbol) [0x0017C922]\n\t(No symbol) [0x0016A536]\n\t(No symbol) [0x001482DC]\n\t(No symbol) [0x001493DD]\n\tGetHandleVerifier [0x004EAABD+2539405]\n\tGetHandleVerifier [0x0052A78F+2800735]\n\tGetHandleVerifier [0x0052456C+2775612]\n\tGetHandleVerifier [0x003151E0+616112]\n\t(No symbol) [0x00225F8C]\n\t(No symbol) [0x00222328]\n\t(No symbol) [0x0022240B]\n\t(No symbol) [0x00214FF7]\n\tBaseThreadInitThunk [0x76B87D59+25]\n\tRtlInitializeExceptionChain [0x77DCB79B+107]\n\tRtlClearBits [0x77DCB71F+191]\n\t(No symbol) [0x00000000]\n\n", "trace": "  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\loginsteps.py\", line 27, in step_impl\n    text = context.driver.find_element(By.XPATH,\"/html/body/div/div/div[2]/nav/div[3]/h1\").text\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1690138046760, "stop": 1690138046779}], "start": 1690138043737, "stop": 1690138046780, "uuid": "d8985a90-d98d-4492-97dc-8309cc89917d", "historyId": "2b075725602fad945d7ecdc80b520fba", "fullName": "Login: Login Authentication, single paramters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}